src/core/field/Opcode.cpp

```diff
@@ -263,7 +263,7 @@ bool Opcode::searchVar(quint8 bank, quint16 address, Operation operation, int va
 			}
 		} else {
 			// Every write vars
-			for (const FF7Var &var : vars) {
+			for (const FF7Var &var : qAsConst(vars)) {
```

Comment:
already done since var is const

Reply:
Line 227  ```QList<FF7Var> vars;```
This is one is not a const

Reply:
Yeah, but there no need to set vars as const if var is not modified

Reply:
Vars is set as a const to prevent possible detachment during the loop.